native/src/components/List.tsx

```diff
@@ -11,10 +11,12 @@ const NoItemsMessage = styled.Text`
 
 type ListProps<T> = {
   items: Array<T>
-  noItemsMessage: ReactElement | string
+  noItemsMessage?: ReactElement | string
```

Comment:
Why have you changed this to be optional?

Reply:
because there is always a ts error saying that this prop is missing or should I include it everywhere?